didact/camelk/first-integration.md

```diff
@@ -0,0 +1,109 @@
+| [Minikube is accessible and running at the command line](didact://?commandId=vscode.didact.requirementCheck&text=minikube-requirements-status$$minikube%20status$$host:%20Running&completion=Minikube%20is%20available%20on%20this%20system. "Tests to see if `minikube status` returns a result"){.didact} 	| *Status: unknown*{#minikube-requirements-status} 	| See [Installing Camel K on Minikube](https://camel.apache.org/camel-k/latest/installation/minikube.html "Documentation on how to Install Apache Camel K on Minikube, with links to the official doc for Minikube installation")
+| [VS Code Extension Pack for Apache Camel by Red Hat is installed](didact://?commandId=vscode.didact.extensionRequirementCheck&text=extension-requirement-status$$redhat.apache-camel-extension-pack&completion=Camel%20extension%20pack%20available. "Checks the VS Code workspace to make sure the extension pack is installed"){.didact} | *Status: unknown*{#extension-requirement-status} 	| [Click here to install](vscode:extension/redhat.apache-camel-extension-pack "Opens the extension page and provides an install link") |
+| [VS Code Tooling for Apache Camel K by Red Hat is installed](didact://?commandId=vscode.didact.extensionRequirementCheck&text=camelk-extension-requirement-status$$redhat.vscode-camelk&completion=Camel%20K%20extension%20pack%20available. "Checks the VS Code workspace to make sure the extension pack is installed"){.didact} | *Status: unknown*{#camelk-extension-requirement-status} 	| [Click here to install](vscode:extension/redhat.vscode-camelk "Opens the extension page and provides an install link") |
+
+## Your First Camel K Integration
+
+You can write an integration in one of several languages supported ([Groovy, Kotlin, JavaScript, Java, XML, etc.](https://camel.apache.org/camel-k/latest/languages/languages.html)), but today we're going to focus on Groovy which is one of the easiest and most popular.
+
+### Step 1: Creating a Folder and Your Integration
+
+First, we need to set up our sample project. Camel K has many examples available at their [GitHub project hosting the source](https://github.com/apache/camel-k/tree/master/examples), but we're just going to use the simplest one: `simple.groovy`.
+
+You can download the file yourself from the GitHub repo and then copy it into a folder in your workspace. Or you can create a sample folder and copy in the file with the link below.
+
+- [ ] [Create a sample folder in your VS Code workspace and we will define it there.](didact://?commandId=vscode.didact.scaffoldProject&extFilePath=redhat.vscode-camelk/didact/camelk/simple-groovy-project.json&completion=Created%20simple-groovy%20project. "Creates a folder and copies in simple.groovy"){.didact}
```

Comment:
this action did nothing for me when I add several folders/files in the workspace

Reply:
I don't believe that is the case, though it may have been difficult to tell at the time. I suspect that the folder/file structure was created beneath the first folder in the workspace whether you noticed it or not.